python/py-photomosaic/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        worldveil photomosaic d24fb85
+version             2020.06.03
+revision            0
+name                py-${github.project}
+categories-append   graphics
+
+license             none
```

Comment:
this isn't a valid license for MacPorts (all license should are present [here](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl)). If there is no license it perhaps could be `public-domain`, but please check with upstream.

Reply:
Thanks, done.

Reply:
Also see https://github.com/worldveil/photomosaic/issues/5